app/controllers/ticket_controller.rb

```diff
@@ -29,8 +31,8 @@ def new
   end
 
   def create
-    @link = Movie.where(id: params[:movie_id], cinema_hall_id: params[:cinema_hall_id])
-    throw :abort unless @link 
+    movie = Movie.find_by(id: params[:movie_id], cinema_hall_id: params[:cinema_hall_id])
+    throw :abort unless movie 
```

Comment:
Consider raising here error which in result return 422 response.

Reply:
@wmaciejak Ok, is everything else good?